### DISCUSSION TITLE
请问有人在服务器上稳定部署成功了吗？

### DISCUSSION THREAD

**Discussion by LL1024** [Mrs4s/go-cqhttp#1721] [Q&A]
我百度云和华为云的服务器都尝试过了，目前都没办法稳定部署。有人部署成功了吗？

描述下我目前的情况：
使用的是最新版本v1.0.0-rc3，服务器为centOS8 64bit，安装的go-cqhttp_1.0.0-rc3_linux_amd64.rpm这个软件包

安装完成之后使用go-cqhttp -d启动机器人，使用的config.yaml/device.json/session.token都是本地测试好生成的文件上传的

机器人启动是没问题的，能正常接收消息，但是没法稳定接受消息，一般短则几分钟十几分钟，长则三个小时左右，机器人【cqhttp这一端】就无法接受消息。

但是考虑到在服务器上部署应该是一个比较常见的场景，而我在discussions和issues都没有找到相关的话题，应该是我部署的哪里有问题，所以在这里请教一下大家～
也提了一个issue，欢迎大家查看 https://github.com/Mrs4s/go-cqhttp/issues/1711

**Comment by Capsule7446** [comment]
如果你愿意使用Docker部署的话,我可以提供我的DockerFile和Docker-compose.yml
